homeassistant/components/caldav/calendar.py

```diff
@@ -80,7 +82,8 @@ def setup_platform(
     days = config[CONF_DAYS]
 
     client = caldav.DAVClient(
-        url, None, username, password, ssl_verify_cert=config[CONF_VERIFY_SSL]
+        url, None, username, password, ssl_verify_cert=config[CONF_VERIFY_SSL],
+        ssl_cert=config[CONF_SSL_CLIENT_CERT]
```

Comment:
I know you'll probably put this in the documentation page, but it would be good to have some comment indicate what value is expected here. In digging I found that this passes through to the [Requests library](https://requests.readthedocs.io/en/latest/user/advanced/#client-side-certificates) and appears to expect either a tuple or a single string path to a certificate file.

In this case, it'd be a single path I suppose and would need to include the private key and the certificate.

Reply:
Yes, its the path to a file. I created a PR for the documentation here https://github.com/home-assistant/home-assistant.io/pull/24127.

There is also a feature request here: https://community.home-assistant.io/t/support-ssl-mutual-authentication-in-caldav-component/459155 where I talk about the change.